src/fundus/publishers/de/__init__.py

```diff
@@ -30,6 +31,13 @@
 
 # noinspection PyPep8Naming
 class DE(PublisherEnum):
+    NetzpolitikOrg = PublisherSpec(
+        name="netzpolitik.org",
+        domain="https://netzpolitik.org/",
+        sources=[Sitemap("https://netzpolitik.org/sitemap.xml"), RSSFeed("https://netzpolitik.org/feed/")],
```

Comment:
Could you add a filter to the sitemap by utilizing the `sitemap_filter` parameter of `Sitemap` to request only pages of type `sitemap_posttype_post`. You can do this by utilizing `regex_filter` and `inverse` from the `fundus.scraping.filter` module.

Reply:
Did you mean `sitemap-posttype-post` instead of `sitemap_posttype_post`?